pandas/tests/io/parser/test_python_parser_only.py

```diff
@@ -314,3 +316,15 @@ def test_malformed_skipfooter(python_parser_only):
     msg = "Expected 3 fields in line 4, saw 5"
     with pytest.raises(ParserError, match=msg):
         parser.read_csv(StringIO(data), header=1, comment="#", skipfooter=1)
+
+
+def test_file_descriptor_leak(python_parser_only):
```

Comment:
* Can't we also test this against the C parser?

Reply:
No problem, done.